release-engineering/role-handbooks/branch-manager.md

```diff
@@ -235,133 +224,70 @@ After having thoroughly read the section on cutting a release version of the han
+Any `krel gcbmgr` command without the `--nomock` flag is a dry run. It is highly encouraged to dry run first before letting `krel gcbmgr` take any actual impact on the release process. All this mock building/releasing can help you verify that you have a working setup!
 
-For example, to execute an actual run:
-
-```shell
-./gcbmgr release master --buildversion=v1.16.0-alpha.0.N+commit-hash --nomock
-```
+To get more information about this, please [see](https://github.com/kubernetes/release/blob/master/docs/krel/gcbmgr.md#important-notes)
 
 **n.b. This run may fail. Mock builds can only be mock released. A nomock release requires a nomock build to be staged.**
 
-Builds against the `master` branch are implicitly the next alpha. `gcbmgr` and `anago` automatically finds and increments the current build number.
+Builds against the `master` branch are implicitly the next alpha. `krel gcbmgr` and `anago` automatically finds and increments the current build number.
 
 ### Beta Releases
 
-Builds against a `release-x.y` branch are implicitly a next beta. `gcbmgr` and `anago` automatically find and increment the current build number.
+Builds against a `release-x.y` branch are implicitly a next beta. `krel gcbmgr` and `anago` automatically find and increment the current build number.
 
 **n.b. If this is a `beta.0` release, there are additional tasks to complete. Please review them _COMPLETELY_ in the [Branch Creation section](#branch-creation), _before_ continuing.**
 
 The command example below is to stage a build for a beta release:
 
-```shell
-./gcbmgr stage release-1.16 --buildversion=v1.16.0-alpha.3.7+b3bde1fe32dd4a --nomock
-```
-
-or
 
-```shell
-./gcbmgr stage release-1.16 --buildversion=$(curl -Ls https://dl.k8s.io/ci/latest.txt) --nomock
-```
+To run the gcbmgr to stage a Beta release [see](https://github.com/kubernetes/release/blob/master/docs/krel/gcbmgr.md#beta-stage)
 
-To publish (release) the build artifacts from staging beta for example, run:
-
-```shell
-./gcbmgr release release-1.16 --buildversion=v1.16.0-alpha.3.N+commit-hash --nomock
-```
+And to  run the gcbmgr to release a Beta release [see](https://github.com/kubernetes/release/blob/master/docs/krel/gcbmgr.md#beta-release)
 
 **n.b. If this is a `beta.0` release, there are additional tasks to complete after the release branch is cut. Please review them _COMPLETELY_ in the [Branch Creation section](#branch-creation).**
 
 ### Release Candidates
 
-Adding the `--rc` flag switches behavior on to building release candidates. Again `gcbmgr` and `anago` automatically finds and increments the current build number.
-
-For example:
-
-```shell
-./gcbmgr stage release-1.16 --rc --buildversion=v1.16.0-beta.2.7+d17cd235699328 --nomock
-```
-
-or
+Adding the `--rc` flag switches behavior on to building release candidates. Again `krel gcbmgr` and `anago` automatically finds and increments the current build number.
```

Comment:
This is now:
`--type string            Release type, must be one of: 'alpha', 'beta', 'rc', 'official' (default "alpha")`